x/marker/types/denom.go

```diff
@@ -25,6 +33,7 @@ const maxDenomMetadataDescriptionLength = 200
 //  - All Denomination unit Aliases have the same SI prefix as their Denom (but maybe different forms, e.g. name vs symbol)
 //  - All Denomination unit Exponents are {SI prefix exponent of the Denom} - {SI prefix exponent of the base}.
 func ValidateDenomMetadataBasic(md banktypes.Metadata) error {
+	sdk.SetCoinDenomRegex(SdkCoinDenomRegex)
```

Comment:
This must be done in app.go not the module or the rest of the modules will reject the changes.

Reply:
Thanks, it didn't feel right putting it there.